concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,170 @@
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ deployment_values }}/var-jenkins.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Clone concent repositories
+          git:
+            repo:    "{{ item.url }}"
+            dest:    "{{ jenkins_home_dir }}/{{ item.name }}"
+            clone:   yes
+            update:  yes
+            version: CI
+            force:   yes
+          with_items:
+          #  - { url: "{{ repositories['concent-deployment'].url }}", name: concent-deployment }
+            - { url: "https://github.com/bartoszbetka/concent-deployment.git", name: concent-deployment }
+            - { url: "{{ repositories['concent-deployment-values'].url }}", name: concent-deployment-values }
+
+        - name: Create directory for concent jenkins secrets
+          file:
+            path:  "{{ jenkins_home_dir }}/{{ item }}"
+            state: directory
+          with_items:
+            - concent-secrets
+            - concent-secrets/{{ cluster }}/
+            - concent-secrets/cloud/
+            - concent-secrets/jenkins/
+
+    - become:      yes
+      become_user: root
+      block:
+        - name: Upload secrets
+          copy:
+            src:   "{{ local_secret_dir }}/{{ cluster }}/{{ item }}"
+            dest:  "{{ jenkins_home_dir }}/concent-secrets/{{ cluster }}/{{ item }}"
+            owner: jenkins
+            group: jenkins
+            mode:  0400
+          with_items:
+            - cluster-secrets.yml
+            - "../cloud/{{ gke.service_account_name }}-private-key.json"
+            - nginx-proxy-ssl.crt
+            - nginx-proxy-ssl.key
+            - nginx-storage-ssl.crt
+            - nginx-storage-ssl.key
+            - "../jenkins/.token"
+
+        - name: Delete default user settings
+          file:
+            path:  /var/lib/jenkins/users/users.xml
+            state: absent
+
+        - name: Create directory for jenkins job and workspace
+          file:
+            path:  "{{ item }}"
+            state: directory
+            owner: jenkins
+            group: jenkins
+          with_items:
+            - "/var/lib/jenkins/jobs/Build-deploy-and-test/"
+            - "/var/lib/jenkins/workspace/"
+            - "/var/lib/jenkins/workspace/Build-deploy-and-test/"
+            - "/var/lib/jenkins/users/admin/"
+            - "/var/lib/jenkins/secrets/"
+
+        - name: Upload jenkins secrets
+          copy:
+            src:   "{{ local_secret_dir }}/jenkins/{{ item.name }}"
+            dest:  "/var/lib/jenkins{{ item.path }}{{ item.name }}"
+            owner: "jenkins"
+            group: "jenkins"
+          with_items:
+            - { path: "/users/admin/", name: "config.xml" }
+            - { path: "/users/admin/", name: "apiTokenStats.xml" }
+            - { path: "/", name: "credentials.xml" }
+            - { path: "/secrets/", name: "master.key" }
```

Comment:
I think it would be better to have same subdirectories in our secret dir. E.g. store `config.xml` in `secrets/jenkins/users/admin/` rather than in `secrets/jenkins/`.

Then you would be able to simplify this to:
``` yaml
            ...
            src:   "{{ local_secret_dir }}/jenkins/{{ item }}"
            dest:  "/var/lib/jenkins/{{ item }}"
            ...
          with_items:
            - users/admin/config.xml
            - users/admin/apiTokenStats.xml
            - credentials.xml
           ...
```